### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by Cadiducho**

### CODE REVIEW

**Comment by fess89** [comment]
what about #242 ?

**Comment by chrosey** [comment]
please check #242 as stated also spanish translations

**Comment by Cadiducho** [comment]
i didn't see it due his name...

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    <string name="abra"></string>
+    <string name="action_sign_in">Iniciar sesión</string>
+    <string name="allow">Permitir</string>
+    <string name="app_settings">Ajustes</string>
+    <string name="enable_location_permission_message">Por favor, activa los permisos de localización para usar esta aplicación</string>
+    <string name="enable_location_permission_title">Activar permisos de localización</string>
+    <string name="error_field_required">Campo requerido</string>
+    <string name="expiring_in">\"Expira en: \"</string>
+    <string name="google_login_failed">Inicio de sesión con Google fallido</string>
+    <string name="login_warning">Es recomendable [...]</string>
```
Comment (socrates2510) [inline app/src/main/res/values-es/strings.xml:16]: Where is the rest of the disclaimer

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <string name="login_warning_title">Atención:</string>
+    <string name="logout_prompt_message">¿Estás seguro de que quieres cerrar sesión?</string>
+    <string name="minutes">\"minutos\"</string>
+    <string name="notification_service_pokemon_in_area">\" Pokémon en area:\"</string>
+    <string name="notification_service_pokemon_near">\" Pokémon cerca.\"</string>
+    <string name="notification_service_scanning">Escanenado</string>
+    <string name="notification_service_stop">Parar servicio</string>
+    <string name="permition_message">Por favor, activa el uso de GPS para usar esta aplicación</string>
+    <string name="meters">metros</string>
+    <string name="pokemon_disappearing_in">\"Desaparece en: \"</string>
```
Comment (socrates2510) [inline app/src/main/res/values-es/strings.xml:26]: Missing the rest of the format %1$02d

**Pull request closed without merging**